src/core/ext/filters/client_channel/lb_policy/subchannel_list.h

```diff
@@ -365,7 +365,7 @@ SubchannelList<SubchannelListType, SubchannelDataType>::SubchannelList(
   // Create a subchannel for each address.
   for (const ServerAddress& address : addresses) {
     RefCountedPtr<SubchannelInterface> subchannel =
-        helper->CreateSubchannel(std::move(address), args);
+        helper->CreateSubchannel(address, args);
```

Comment:
Please also change line 366 from `const ServerAddress&` to `ServerAddress`, and change line 386 to use `std::move(address)`.

Reply:
Done!